frontend/components/ClubEditPage/ClubEditCard.tsx

```diff
@@ -493,7 +493,7 @@ export default function ClubEditCard({
           type: 'multiselect',
           placeholder: `Select schools relevant to your ${OBJECT_NAME_SINGULAR}!`,
           choices: schools,
-          hidden: !showTargetFields,
+          hidden: schools.length === 0 || !showTargetFields,
```

Comment:
Instead of doing this, you can take advantage of how all the other fields are shown or hidden in `branding.tsx`. See how some fields are hidden or shown based on the variables in that file.

This allows all of our differences in behavior to be standardized.